### PR TITLE
Hotfix: style, intents

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Debugging Sendlimiter in prod.

### CODE REVIEW

**Comment by EthanThatOneKid** [comment]
Follow-up of #1.

File: main.go [hunk 1]
```diff
@@ -72,6 +72,9 @@ func main() {
 
 	// Add the needed Gateway intents.
 	s.AddIntents(gateway.IntentGuildMessages)
+	s.AddIntents(gateway.IntentMessageContent)
+	s.AddIntents(gateway.IntentDirectMessages)
```
Comment (diamondburned) [inline main.go]: This is only needed for receiving DM events. It's not needed for sending DMs.

File: main.go [hunk 2]
```diff
@@ -72,6 +72,9 @@ func main() {
...
+	s.AddIntents(gateway.IntentGuildMessages)
```
Comment (diamondburned) [inline main.go]: Added twice.

File: main.go (hunk 1 above)
Comment (EthanThatOneKid) [reply main.go]: Addressed by <https://github.com/acmcsufoss/sendlimiter/pull/2/commits/45c33f9a01bb1893c644ca852d56241ec720913c>.

File: main.go (hunk 2 above)
Comment (EthanThatOneKid) [reply main.go]: Good catch; thanks for the review! Addressed in <https://github.com/acmcsufoss/sendlimiter/pull/2/commits/45c33f9a01bb1893c644ca852d56241ec720913c>.

**Pull request merged**